CMakeLists.txt

```diff
@@ -40,10 +40,11 @@ endif()
 option(JSON_BuildTests                     "Build the unit tests when BUILD_TESTING is enabled." ${JSON_BuildTests_INIT})
 option(JSON_CI                             "Enable CI build targets." OFF)
 option(JSON_Diagnostics                    "Use extended diagnostic messages." OFF)
+option(JSON_DisableEnumSerialization       "Disable default integer enum serialization." OFF)
```

Comment:
Oops. I accidentally duplicated that line when I manually reverted your changes. 
```suggestion
```
(To Niels: I reverted the re-ordering to avoid merge conflicts. I'll clean that section up as part of adding dependent CMake options and CMake feature summary.)